src/Logging/RpcLogger.cs

```diff
@@ -14,13 +15,15 @@ namespace Microsoft.Azure.Functions.PowerShellWorker.Utility
     internal class RpcLogger : ILogger, IDisposable
     {
         private const string SystemLogPrefix = "LanguageWorkerConsoleLog";
-        private MessagingStream _msgStream;
+        private readonly MessagingStream _msgStream;
+        private readonly StringBuilder _systemLogMsg;
```

Comment:
Is there a reason you went with having a StringBuilder property rather than just instantiating a new StringBuilder? 

Is StringBuilder construction expensive compared to Clearing the one every time you log?

Reply:
It's mainly about allocating the memory for the `StringBuilder`. A new instance means new memory allocation, while `Clean` only set the length of the `StringBuilder` to 0, the allocated memory remains.

Reply:
Sounds good.